tests/fixtures/bidaf/data/data.json

```diff
@@ -0,0 +1,120 @@
+{
```

Comment:
We have `tests/fixtures/squad_example.json`.  This has more examples than that, but we probably don't need both.  Maybe move this to `tests/fixtures/data/squad.json`, or `tests/fixtures/data/squad/train.json`, or something, and get rid of the other one?  I think having a `tests/fixtures/data` base directory for data files is better than putting this under `tests/fixtures/bidaf/`, because the dataset reader also uses the data file, or other models that use squad.